projects/kit/components/calendar-range/calendar-range.component.ts

```diff
@@ -114,13 +114,15 @@ export class TuiCalendarRange implements OnInit, OnChanges {
     }
 
     public get computedMonth(): TuiMonth {
-        return this.value ? this.value.from : this.defaultViewedMonth;
+        return (
+            (this.items.length ? this.value?.to : this.value?.from) ||
+            this.defaultViewedMonth
+        );
     }
 
+    // TODO: refactor using only getter or ngOnChanges
     public ngOnChanges(): void {
```

Comment:
perhaps `ngOnChanges` is not needed, but since it was already there, I not touch it